spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -359,6 +392,21 @@
           end
         end
       end
+
+      context 'when the user chooses to remember their device' do
+        it 'does not save a remember_device cookie' do
```

Comment:
Is the title of this test right?

Reply:
Yeah, I think it could be named better perhaps. This is nested under the confirmation context.

One thing to check with @micahtaylor about is whether the "remember device" option should be available during device confirmation?

Reply:
@andrewhughey and I went back and forth between that and landed on this page - https://cm-jira.usa.gov/secure/attachment/55472/03e%20-%202fa%20-%20enter%20code%20%26%20remember%20device.png

Reply:
@micahtaylor: Yeah, so that page is reused for sign in, sign up, and when the user changes their phone. My question is about which of those screens we want the option on, or do we want it on all of them?

Reply:
Ah, right. At the very least sign in. Depending on if there is any major level of effort it would be nice to have it on all 3.

Reply:
Right now I only have it on sign in, but I can do all 3.